drivers/adc/ad463x/ad463x.c

```diff
@@ -53,6 +53,19 @@
 /******************************************************************************/
 #define AD463x_TEST_DATA 0xAA
 
+#define NANO 1000000000ULL
+#define ADAQ4224_GAIN_MAX_NANO 6670000000ULL
+
+/*
+ * Gains computed as fractions of 1000 so they can be expressed by integers.
+ */
+static const int32_t ad4630_gains[4] = {
+	330, 560, 2220, 6670
+};
+
+/* Declaration for DIV_ROUND_CLOSEST_ULL if not provided by the system */
+#define DIV_ROUND_CLOSEST_ULL(x, divisor) (((x) + ((divisor) / 2)) / (divisor))
```

Comment:
same comment about this macro:
https://github.com/analogdevicesinc/no-OS/blob/main/include/no_os_util.h#L60